stable/fluentd/templates/deployment.yaml

```diff
@@ -89,6 +89,7 @@ spec:
           mountPath: /etc/fluent/config.d
         - name: buffer
           mountPath: "/var/log/fluentd-buffers"
+      serviceAccountName: {{ template "fluentd.fullname" . }}
```

Comment:
See https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md for howto do this properly.

Reply:
Thanks I updated the PR